app/code/GorbanSv/MakeYourStyle/view/frontend/web/css/source/_module.less

```diff
@@ -0,0 +1,15 @@
+//
+//  Dropdowns variables
+//  _____________________________________________
+
+@dropdown-list__border: 3px solid #006400;
```

Comment:
This variable is used in other .less files. Should use another name for variable and pass it to the mixin